Tools-Override/FrameworkTargeting.targets

```diff
@@ -263,8 +263,8 @@
     </ItemGroup>
 
     <PropertyGroup>
-      <_propsFilename>$(TargetName)</_propsFilename>
-      <_propsFilename Condition="'$(_propsFilename)' == ''">$(MSBuildProjectName)</_propsFilename>
+      <_propsFilename>$(TargetName).$(TargetGroup)-$(OSGroup)</_propsFilename>
```

Comment:
Why not Configuration? That allows someone to build both Debug and Release as well (or you can do that at a higher level).